### ISSUE TITLE
FLINT version macro needs update

### ISSUE THREAD

**Issue by advanpix** [wbhart/flint2#1073]
FLINT version macro needs update (it still shows 2.8.0):

```
#define __FLINT_VERSION 2
#define __FLINT_VERSION_MINOR 8
#define __FLINT_VERSION_PATCHLEVEL 0
#define FLINT_VERSION "2.8.0"
```

**Comment by wbhart** [comment]
Thanks. I have no idea how that's been overlooked for four patch releases. It almost seems not possible, but this is what GitHub is showing. It hasn't been updated for 14 months.

**Comment by advanpix** [comment]
Absolutely, I couldn't believe my eyes too. I noticed this because my build script gets version from these macro values.

**Comment by albinahlback** [comment]
This can be closed due to [a9c3f41](https://github.com/wbhart/flint2/commit/a9c3f41a70aae7ae229e11286543ea139df8df3f).

**Comment by wbhart** [comment]
I haven't resolved the issue on the 2.8 branch yet.
